### PR TITLE
CIRA Support

### PR DESCRIPTION
**Pull request by BlackthornYugen**
Adding support for CIRA re: GH-8

### CODE REVIEW

**Comment by BlackthornYugen** [comment]
Made an issue on a [different repo by mistake](https://github.com/picatz/doh/issues/8)... I guess I can't actually make an issue on this repo.

**Comment by comzeradd** [comment]
Hi @BlackthornYugen, thanks for the contribution :+1: 

Can you rebase your branch to resolve the conflicts?

**Comment by BlackthornYugen** [comment]
Sure thing! Glad to see that custom DNS entries are supported now. The join for choices was something I considered doing in this change but I was too lazy. 

I tested that `a2bb368` seems to work:

```
(venv) jsteel@blacktop:~/git/doh-cli$ git log --all --decorate --oneline --graph --max-count=10 # git log a dog
* a2bb368 (HEAD -> CIRA-8, yugen/CIRA-8) 8 - CIRA Support
* ea8be95 (origin/master, origin/HEAD) Support custom DoH endpoints, more verbose debug
| * 19a4371 (yugen/AnyRecordType, AnyRecordType) Allow user to use any type (IE: MX Records)
| * 789f5b1 #8 - CIRA Support: Fix arguments
| * c8c6139 #8 - CIRA Support
|/  
* 9c001f2 (yugen/master, master) Add author email
* bea9543 Add long description content type
* 5c5908c Initial commit
(venv) jsteel@blacktop:~/git/doh-cli$ date
Sat Mar 28 23:02:34 EDT 2020
(venv) jsteel@blacktop:~/git/doh-cli$ python ./setup.py install | tail
warning: no files found matching '*' under directory 'doh-cli'
Installing chardetect script to /home/jsteel/git/doh-cli/venv/bin

Using /home/jsteel/git/doh-cli/venv/lib/python3.7/site-packages/chardet-3.0.4-py3.7.egg
Searching for certifi==2019.11.28
Best match: certifi 2019.11.28
Processing certifi-2019.11.28-py3.7.egg
certifi 2019.11.28 is already the active version in easy-install.pth

Using /home/jsteel/git/doh-cli/venv/lib/python3.7/site-packages/certifi-2019.11.28-py3.7.egg
Finished processing dependencies for doh-cli==0.1
(venv) jsteel@blacktop:~/git/doh-cli$ doh-cli --dns cira steelcomputers.com A
[{"Query": "steelcomputers.com.", "TTL": "212", "RR": "A", "Answer": "144.217.207.193"}]
```

File: bin/doh-cli [hunk 1]
```diff
@@ -14,6 +14,13 @@ url = {
     "cloudflare":    "https://cloudflare-dns.com/dns-query",
     "quad9":         "https://dns.quad9.net/dns-query",
     "cleanbrowsing": "https://doh.cleanbrowsing.org/doh/family-filter/?ct",
+    # CIRA's Canadian Shield
+    # - "Private is a free DNS resolution service"
+    # - "Protected adds malware and phishing blocking"
+    # - "Family blocks malware and phishing plus pornographic content"
+    "cira":          "https://private.canadianshield.cira.ca/dns-query",
+    "cira-protect":  "https://protected.canadianshield.cira.ca/dns-query",
+    "cira-family":   "https://protected.canadianshield.cira.ca/dns-query",
```
Comment (comzeradd) [inline bin/doh-cli]: Shouldn't that url be different than the one above?

Comment (BlackthornYugen) [reply bin/doh-cli]: Oh, good catch! Should be family.canadianshield.

Comment (BlackthornYugen) [reply bin/doh-cli]: Fixed in 52bc372!

```
(venv) jsteel@blacktop:~/git/doh-cli$ git log --all --decorate --oneline --graph --max-count=10 # git log a dog
* 52bc372 (HEAD -> CIRA-8, yugen/CIRA-8) 8 - CIRA Support
| * 2975c8b (yugen/AnyRecordType, AnyRecordType) Allow user to use any type (IE: MX Records)
|/  
* ea8be95 (origin/master, origin/HEAD) Support custom DoH endpoints, more verbose debug
* 9c001f2 (yugen/master, master) Add author email
* bea9543 Add long description content type
* 5c5908c Initial commit

(venv) jsteel@blacktop:~/git/doh-cli$ date
Sun Mar 29 11:04:05 EDT 2020

(venv) jsteel@blacktop:~/git/doh-cli$ curl http://xtube.com --resolve xtube.com:80:$(doh-cli --dns cira xtube.com A | jq '.[0].Answer' -r) -sD /dev/stderr > /dev/null
HTTP/1.1 301 Moved Permanently
Content-length: 0
Location: https://www.xtube.com/

(venv) jsteel@blacktop:~/git/doh-cli$ curl http://xtube.com --resolve xtube.com:80:$(doh-cli --dns cira-family xtube.com A | jq '.[0].Answer' -r) -sD /dev/stderr > /dev/null
HTTP/1.1 302 Moved Temporarily
Server: awselb/2.0
Date: Sun, 29 Mar 2020 15:04:14 GMT
Content-Type: text/html
Content-Length: 126
Connection: keep-alive
Location: https://cira.ca:443/CanadianShield/Active/FamilyFilter
```

**Pull request merged**